drivers/i2s/i2s_nrf_tdm.c

```diff
@@ -480,12 +482,14 @@ static int tdm_nrf_configure(const struct device *dev, enum i2s_dir dir,
 
 	__ASSERT_NO_MSG(tdm_cfg->mem_slab != NULL && tdm_cfg->block_size != 0);
 
-	if ((tdm_cfg->block_size % sizeof(uint32_t)) != 0 ||
+#if NRF_ERRATA_STATIC_CHECK(54H, 39)
+	if (NRF_ERRATA_DYNAMIC_CHECK(54H, 39) || (tdm_cfg->block_size % sizeof(uint32_t)) != 0 ||
```

Comment:
shouldn't be `NRF_ERRATA_DYNAMIC_CHECK(54H, 39) && ` ? If errata does not apply we shouldn't check for the second condition. Same goes for `tdm_nrf_write` change